dts/bindings/power/state.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Properties for power management state
+
+compatible: "pm-state"
+
+properties:
+    pm-state:
+        type: string
+        required: true
+        description: indicates a power state
+        enum:
```

Comment:
Based on https://github.com/zephyrproject-rtos/zephyr/pull/30406#discussion_r535346182 I'm expecting the tokenizing support to allow these to be changed to lower case with hyphen separators.  That does not have to be done now, but should be done as soon as the tooling supports it.

Reply:
> Based on [#30406 (comment)](https://github.com/zephyrproject-rtos/zephyr/pull/30406#discussion_r535346182) I'm expecting the tokenizing support to allow these to be changed to lower case with hyphen separators

That is indeed the case with the latest update, just pushed